oat-sa/ckeditor-dev
# Fix BSA 183 - Add missing translations plugins

**Related to**

https://oat-sa.atlassian.net/browse/BSA-183

**Description**

This PR is to allow dynamic translations of CKEditor based on selected language by the user

**AC**

- Translate only `CKeditor` keys in NL-DE-FR languages

**How to test it**

- `git pull` and run script `dev/builder/build.sh`
- Copy the folder `dev/builder/release/ckeditor` created
- Paste in your local env in `tao/views/node_modules/@oat-sa/tao-core-shared-libs/lib/ckeditor`
- Compile `tao` extension with `npx grunt taobundle` in `tao/views/build` If the process give you error about missing Grunt, `npm i` first
- Remove cache folder in `data/generis/cache`
- Open local environment in private mode for avoid issues with browser extensions, cookies and more browser stuff related
- Go to settings of the user, and change to French for example
- Create an item with an extended text interaction inside. Click to see the `ckeditor` and hover with the cursor each button.
